internal/host/pool_host_statemachine.go

```diff
@@ -70,6 +70,24 @@ func NewPoolHostStateMachine(sm stateswitch.StateMachine, th *transitionHandler)
 		PostTransition:   th.PostBindHost,
 	})
 
+	// Unbind host
+	sm.AddTransition(stateswitch.TransitionRule{
```

Comment:
This transaction should be in the internal/host/statemachine.go file, since it is being executed on the host belonging to a cluster

Reply:
done